tests/components/concurrency.test.ts

```diff
@@ -3696,6 +3696,42 @@ test("another scenario with delayed rendering", async () => {
   ]).toBeLogged();
 });
 
+test.only("delayed fiber does not get rendered if it was cancelled", async () => {
```

Comment:
:-)

Reply:
(starting to think we really need something to check this in CI)